crates/matrix-sdk/Cargo.toml

```diff
@@ -85,7 +85,7 @@ matrix-sdk-sled = { version = "0.2.0", path = "../matrix-sdk-sled", default-feat
 mime = "0.3.16"
 rand = { version = "0.8.5", optional = true }
 reqwest = { version = "0.11.10", default_features = false }
-ruma = { workspace = true, features = ["compat", "rand", "unstable-msc2448", "unstable-msc2965"] }
+ruma = { workspace = true, features = ["compat", "rand", "unstable-msc2448", "unstable-msc2965", "unstable-msc3575"] }
```

Comment:
why activate that feature always? Isn't `matrix-sdk-base/experimental-sliding-sync` already activating that properly? MSC3575 should probably not be activated by default yet.